src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/Example2.java

```diff
@@ -0,0 +1,23 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidDoubleBraceInitialization"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.avoiddoublebraceinitialization;
+
+import java.util.ArrayList;
+
+import java.util.List;
+
+// xdoc section -- start
+class Example2 {
+  List<Object> list = new ArrayList<>() { // OK, not pure double brace pattern
```

Comment:
Please do us a favor.
Combine example2 to example1. We just need one example.
`// OK, as it is not double brace pattern`

Reply:
@romani done as requested

Reply:
Please do not resolve conversation, just reply by comment.